index.html

```diff
@@ -79,108 +79,7 @@ <h2 class="social_media">LET’S CONNECT</h2>
-          <h3>2015</h3>
-        </div>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul>
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <div class="action">
-          <button class="btn">
-            See Project
-          </button>
-        </div>
-      </div>
-    </div>
-    <div class="card">
-      <img src="images/Snapshoot Portfolio 2.png" alt="Portfolio image 2">
-      <div class="work_desc">
-        <h2>Multi-Post <br> Stories</h2>
-        <div class="heading_3">
-          <h3 class="main_h3">CANOPY</h3>
-          <span></span>
-          <h3>Back End Dev</h3>
-          <span></span>
-          <h3>2015</h3>
-        </div>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul>
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <div class="action">
-          <button class="btn">
-            See Project
-          </button>
-        </div>
-      </div>
-      <img class="desktop_image" src="images/Portoflio Card/Snapshoot Portfolio_2.png" alt="Portfolio image 2">
-    </div>
-    <div class="card">
-      <img src="images/Snapshoot Portfolio 3.png" alt="Portfolio image 3">
-      <img class="desktop_image" src="images/Portoflio Card/Snapshoot Portfolio_3.png" alt="Portfolio image 3">
-      <div class="work_desc">
-        <h2>Tonic</h2>
-        <div class="heading_3">
-          <h3 class="main_h3">CANOPY</h3>
-          <span></span>
-          <h3>Back End Dev</h3>
-          <span></span>
-          <h3>2015</h3>
-        </div>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul>
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <div class="action">
-          <button class="btn">
-            See Project
-          </button>
-        </div>
-      </div>
-    </div>
-    <div class="card">
-      <img src="images/Snapshoot Portfolio 4.png" alt="Portfolio image 4">
-      <div class="work_desc">
-        <h2>Multi-Post <br> Stories</h2>
-        <div class="heading_3">
-          <h3 class="main_h3">CANOPY</h3>
-          <span></span>
-          <h3>Back End Dev</h3>
-          <span></span>
-          <h3>2015</h3>
-        </div>
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-        <ul>
-          <li>html</li>
-          <li>css</li>
-          <li>javaScript</li>
-        </ul>
-        <div class="action">
-          <button class="btn">
-            See Project
-          </button>
-        </div>
-      </div>
-      <img class="desktop_image" src="images/Portoflio Card/Snapshoot Portfolio_4.png" alt="Portfolio image 4">
-    </div>
-  </section>
+  <section class="works_container"></section>
```

Comment:
Well done overall 💯 but this needs some improvement on the design side, you have some improvement to make on both mobile and Desktop version:
- [ ] The spacing between these elements aren't quite right. Please add some more spacing between these elements to match the Figma. The buttons texts shouldn't be taking two lines and should have a gap between them.
![Screenshot 2023-03-09 at 09 03 12](https://user-images.githubusercontent.com/45739963/224048884-edb75469-b865-4993-b017-7693e16ac245.png)

- [ ] On the mobile version the spacing needs some improvement as well, you can try reduce the margin of the popup for example.

![Screenshot 2023-03-09 at 09 05 13](https://user-images.githubusercontent.com/45739963/224049338-241c8b48-4780-4831-9f64-08cbd8c37fa1.png)
![Screenshot 2023-03-09 at 09 06 11](https://user-images.githubusercontent.com/45739963/224049569-215047da-84a1-4973-b9f8-d1559100efc6.png)

Please adjust the design by making the texts smaller, and adding some more spacing between the elements tp match the given template. 😉